workers.go

```diff
@@ -533,11 +537,16 @@ func (api *API) ListWorkerScripts(ctx context.Context) (WorkerListResponse, erro
 // UploadWorker push raw script content for your worker.
 //
 // API reference: https://api.cloudflare.com/#worker-script-upload-worker
-func (api *API) UploadWorker(ctx context.Context, requestParams *WorkerRequestParams, data string) (WorkerScriptResponse, error) {
+func (api *API) UploadWorker(ctx context.Context, requestParams *WorkerRequestParams, params *WorkerScriptParams) (WorkerScriptResponse, error) {
+	if params.Module {
+		return api.UploadWorkerWithBindings(ctx, requestParams, params)
```

Comment:
module seems to only support the multipart form uploads. Was getting the following error without this.
```
2022/03/17 16:45:12 /accounts/ed3ac05d9853a82d33b54902b8b14c0c/workers/scripts/module-upload-test
2022/03/17 16:45:12 HTTP status 415: workers.api.error.content_type (10001)
```